### PR TITLE
ms17_010_psexec: fix RHOST in "authenticating..." message

### PR DESCRIPTION
**Pull request by cnotin**
The "Authenticating..." message has an info missing:
> Authenticating to  as user 'user'...

It is due to a confusion between `RHOST` and `RHOSTS`.
This module uses `RHOST` as shown just below in the code:
```eternal_pwn(datastore['RHOST'])```

### CODE REVIEW

**Comment by wchen-r7** [comment]
Note: rspec failure resolved in #11269.

**Comment by cnotin** [comment]
Well, I was surprised to see the build fail at first 😉

File: modules/exploits/windows/smb/ms17_010_psexec.rb [hunk 1]
```diff
@@ -103,7 +103,7 @@ def initialize(info = {})
   def exploit
     begin
       if datastore['SMBUser'].present?
-        print_status("Authenticating to #{datastore['RHOSTS']} as user '#{splitname(datastore['SMBUser'])}'...")
+        print_status("Authenticating to #{datastore['RHOST']} as user '#{splitname(datastore['SMBUser'])}'...")
```
Comment (wvu-r7) [inline modules/exploits/windows/smb/ms17_010_psexec.rb]: Is the `rhost` method available in this context?

Comment (cnotin) [reply modules/exploits/windows/smb/ms17_010_psexec.rb]: `datastore['RHOSTS']` (current) is not available. But `datastore['RHOST']` (suggested) is available, as demonstrated two lines below with:
`eternal_pwn(datastore['RHOST'])`

Or do you mean something different with "`rhost` **method**"?

Comment (wvu-r7) [reply modules/exploits/windows/smb/ms17_010_psexec.rb]:
```
msf5 exploit(windows/smb/ms17_010_psexec) > pry
[*] Starting Pry shell...
[*] You are in exploit/windows/smb/ms17_010_psexec

[1] pry(#<Msf::Modules::Exploit__Windows__Smb__Ms17_010_psexec::MetasploitModule>)> method(:rhost).source_location
=> ["/rapid7/metasploit-framework/lib/msf/core/exploit/tcp.rb", 266]
[2] pry(#<Msf::Modules::Exploit__Windows__Smb__Ms17_010_psexec::MetasploitModule>)>
```

Comment (wvu-r7) [reply modules/exploits/windows/smb/ms17_010_psexec.rb]: I don't think it matters much. You'll find more consistency with the datastore, anyway.

**Pull request merged**